MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -1721,12 +1721,6 @@ private void ApplyTranscodingConditions(StreamInfo item, IEnumerable<ProfileCond
                     continue;
                 }
 
-                // No way to express this
-                if (condition.Condition == ProfileConditionType.GreaterThanEqual)
-                {
-                    continue;
-                }
-
```

Comment:
Don't change such behavior in bugfix releases as it might break some weird clients.